content/20.treatments.md

```diff
@@ -856,6 +856,17 @@ A list of current studies and their results, using carefully the information req
 **Summary.**
 Summarize the state of the neutralizing antibody approach.
 -->
+
+##### Adjuvants for Vaccines
+
+Adjuvants include a variety of molecules or larger microbial-related products which sometimes are or include immunostimulants or immunomodulators or more generally have an effect on the immune system or immune responses of interest.
+Adjuvants are sometimes included within vaccines, especially vaccines other than live attenuated and inactivated viruses, in order to enhance the immune response.
+A review on the development of SARS-CoV-2 vaccines [@doi:10.3390/vaccines8020153] also included a brief summary of considerations for adjuvants to research for the vaccines including a brief description of some already commonly used adjuvants.
```

Comment:
```suggestion
A review on the development of SARS-CoV-2 vaccines [@doi:10.3390/vaccines8020153] also included a brief summary of considerations for the inclusion of adjuvants.
```
@yemarshall the original sentence is a bit complex, what about just making the end lead in to the next sentence?

Reply:
the shorter version above loses & changes meaning. Options:
(1) keep as is (preferred b/c verified already)
or
(2) Make the structure sort of simpler by splitting into 2 clauses with semi-colon:
A review on the development of SARS-CoV-2 vaccines [@doi:10.3390/vaccines8020153] also included a brief summary of considerations for adjuvants to research for the vaccines; the summary also included a brief description of some already commonly used adjuvants.